crypto/src/main/scala/org/bitcoins/crypto/ECDigitalSignature.scala

```diff
@@ -137,6 +137,22 @@ object ECDigitalSignature extends Factory[ECDigitalSignature] {
     }
   }
 
+  def fromFrontOfBytes(bytes: ByteVector): ECDigitalSignature = {
```

Comment:
nit: Scaladoc, does it just peal off the first 64 bytes?

Reply:
will do, and no it parses a variable length ECDSA sig